_posts/2015-09-04-channel.html

```diff
@@ -435,6 +435,272 @@
+				<th>parameter</th>
+				<th>type</th>
+				<th>importance</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td><strong>device_name</strong></td>
+				<td>string</td>
+				<td>REQUIRED</td>
+				<td>Name of the device</td>
+			</tr>
+			<tr>
+				<td><strong>valid_from</strong></td>
+				<td>timestamp</td>
+				<td>REQUIRED</td>
+				<td>From that time the user can login</td>
+			</tr>
+			<tr>
+				<td><strong>valid_to</strong></td>
+				<td>timestamp</td>
+				<td>OPTIONAL</td>
+				<td>Until that time the user can login</td>
+			</tr>
+		</table>
+
+
+		<h6>Response</h6>
+
+		<p>All of these elements are containing these informations:</p>
+		<table class="table responsive">
+			<tr>
+				<th>key</th>
+				<th>type</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td>success</td>
+				<td>boolean</td>
+				<td>Method success</td>
+			</tr>
+			<tr>
+				<td>device_name</td>
+				<td>string</td>
+				<td>Name of the device password</td>
+			</tr>
+			<tr>
+				<td>user_name</td>
+				<td>string</td>
+				<td>Username of the device password</td>
+			</tr>
+			<tr>
+				<td>password</td>
+				<td>string</td>
+				<td>Generated password</td>
+			</tr>
+			<tr>
+				<td>valid_from</td>
+				<td>timestamp</td>
+				<td>From that time the user can login</td>
+			</tr>
+			<tr>
+				<td>valid_to</td>
+				<td>timestamp</td>
+				<td>Until that time the user can login</td>
+			</tr>
+		</table>
+
+		<h6>Positive response example</h6>
+
+		<pre><code class="json">{
+  "success": true,
+  "device_name": "Sample Device 4",
+  "user_name": "jgun36pdc79",
+  "password": "jgcxathqhyqcdxp",
+  "valid_from": "1497571200",
+  "valid_to": "1500249600"
+}</code></pre>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>unauthorized</td>
+				<td>401 Unauthorizedt</td>
+				<td>The provided Access Token is missing, revoked, expired or malformed</td>
+			</tr>
+			<tr>
+				<td>input_validation_error</td>
+				<td>400 Bad request</td>
+				<td>If some parameter are missing, or bad request</td>
+			</tr>
+		</table>
+
+		<h3>Delete device password</h3>
+		<p>Edit a channel's title, description and tags.</p>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">DELETE</td>
+				<td><a>https://api.ustream.tv/users/self/deviceuser/DEVICE_NAME.FORMAT</a>format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Parameters</h6>
+		<p>This request has no parameters.</p>
+
+		<h6>Response</h6>
+		<p>On success a response with HTTP status 200 OK is returned.</p>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>input_validation_error</td>
+				<td>400 Bad request</td>
+				<td>If deleting fails</td>
```

Comment:
Nem is tudom, az 500-as errorokat külön dokumenbtáljuk-e, vagy csak központilag. Mindenesetre tényleg jó lenne látni, törlésnél milyen user errort lehet elkövetni, aminek 400 a jutalma.